lib/utilities.js

```diff
@@ -329,6 +329,36 @@ const jsonParse = (buffer) => {
   }
 };
 
+const flatObject = (sourceObj, nameField) => {
+  const targetObj = {};
+
+  for (const [key, value] of Object.entries(sourceObj)) {
+    if (
+      typeof value !== 'object' ||
+      (typeof value === 'object' && Array.isArray(value))
```

Comment:
I suggest extracting it to function 
```suggestion
      !isObject(value)
```

Reply:
@dromix @georgolden 
Done 

and now I have question 
need new func isObject export from utilities 
to use in another files ?